views/partials/filters.hbs

```diff
@@ -29,6 +29,10 @@
 			<div class="ui field input">
 				<input type="text" placeholder="search" id="filter-search">
 			</div>
+			<br/>
+			<div onclick="$('#filter-search').val('.jpeg,.jpg,.png,.gif')" class="ui mini icon button"><i class="file image outline icon"></i> Image</div>
+			<div onclick="$('#filter-search').val('.mp3,.wav,.flac')" class="ui mini icon button"><i class="file audio outline icon"></i> Audio</div>
+			<div onclick="$('#filter-search').val('.mp4,.webm')" class="ui mini icon button"><i class="file video outline icon"></i> Video</div>
```

Comment:
Perhaps wrap these buttons in one element and combine them with the textbox? Or at least something to make them look less cramped up against the textbox

Here's an idea (making the input field 'action mini' and putting buttons inside it)

![](https://shitty.download/Znh2K.png)

```html
			<div class="ui field action input mini">
				<input type="text" placeholder="search" id="filter-search">
				<div onclick="$('#filter-search').val('.jpeg,.jpg,.png,.gif')" class="ui mini icon button"><i class="file image outline icon"></i> Image</div>
				<div onclick="$('#filter-search').val('.mp3,.wav,.flac')" class="ui mini icon button"><i class="file audio outline icon"></i> Audio</div>
				<div onclick="$('#filter-search').val('.mp4,.webm')" class="ui mini icon button"><i class="file video outline icon"></i> Video</div>
			</div>
```